cPP_DSC.adoc

```diff
@@ -356,58 +356,58 @@ One way to categorize keys is by the cryptographic functions they are allowed to
 
 Note that although chained certificates (see certificates below) are one form of a sequence of keys, each of which signs another key, the creation and verification of such a chain of certificates is out of scope for the core requirements of the cPP; however, it may be added as a package if one or both of these features (creating the chain and verifying the chain) is indeed present in the DSC. Nonetheless, the primitives of signing and verification are present due to other cryptographic operations in scope for this cPP.
 
 Intermediate Keys should always be protected (i.e. wrapped) by either a Root Key or another Intermediate Key.
 
-Leaf Objects consist of Authorization Data and Leaf Keys. Leaf Keys can be either signing or encryption keys. Leaf Objects collectively refers to data that should be wrapped by either a Root Key or a KEK and is not subsequently used as a KEK itself. Encryption Leaf Keys do not wrap other keys (at least in the context of the DSC; what happens outside the DSC with Leaf Keys is out of its control). In many contexts, an Encryption Leaf Key is known as a Data Encryption Key (DEK). In the context of the DSC, this cPP will not assume how the user of the DSC will use the Leaf Keys it creates, and will refrain from using the term DEK.
+Leaf Objects consist of Authorization Data and Leaf Keys. Leaf Keys can be either encryption, decryption, signature generation, or signature verification keys. Leaf Objects collectively refers to data that should be wrapped by either a Root Key or a KEK and is not subsequently used as a KEK itself. Leaf Keys used for encryption/decryption do not wrap other keys (at least in the context of the DSC; what happens outside the DSC with Leaf Keys is out of its control). In many contexts, a Leaf Key used for encryption/decryption is known as a Data Encryption Key (DEK). In the context of the DSC, this cPP will not assume how the user of the DSC will use the Leaf Keys it creates, and will refrain from using the term DEK.
 
-Certificates contain either signed public keys, signed encryption/decryption keys, or some sort of Authorization Data. Signature keys come in several varieties: asymmetric signing keys, which contain a private key for signing (and maybe also the public key for verification) and verification keys, which contains only the public verification key and does not contain the private key (and thus cannot perform a signing function). There are also symmetric signature keys. In this case these consist of only a single key for both signing and verifying.
+Certificates contain either signed public keys or some sort of Authorization Data. Signature keys come in several varieties: signature generation keys, which contain a private key for signing (and maybe also the public key for verification) and signature verification keys, which contain only the public verification key and do not contain the private key (and thus cannot perform a signing function).
```

Comment:
I have never heard of a certificate containing encryption/decryption keys, but perhaps that's a lack of knowledge on my part. If that was really intended, I will add it back.
Regarding "symmetric signature keys", is this referring to message authentication codes? It seems a bit strange to discuss that in this paragraph; if a certificate would contain the MAC key, anyone could generate MAC tags.

Reply:
Keep the last sentence that was deleted, but the rest of this suggestion is fine.